client/src/visualiser-src/avl-tree-visualiser/util/codeSnippets.ts

```diff
@@ -12,14 +11,12 @@ export const insertCodeSnippet = `struct node *insert(struct node *node, int val
     node->height = height(node);
     int balance = height(node->left) - height(node->right);
     if (balance > 1) {
-        if (value > node->left->value) {
+        if (value > node->left->value) 
```

Comment:
I'm thinking since we want to have it as c like as we can, maybe we want to have all the conditional lines (if, else, etc) using {}? Just for consistency and accuracy

Reply:
after 3 hours of contemplating my beloved braces god (help me) still decided to remove them: doesn't impact accuracy + it's consistent with 2521 lecture code